cristin-import/src/test/resources/features/DegreePhdRules.feature

```diff
@@ -0,0 +1,9 @@
+Feature:
+  Mapping rules for DegreePhds
+
+  Background:
+    Given a valid Cristin Result with secondary category "DRGRADAVH"
+
+  Scenario: Cristin Result "Ph.D thesis" is converted to an NVA entry with type "DegreePhd".
```

Comment:
Use the description found in the respective file in Confluence. "Ph.D thesis" was an example.   We want to use the file so that there is no ambiguity in our communication with Terje. Although, one can say that we Doctoral thesis or Dissertation or Ph.D. thesis are synonymous, just habits create ambiguity in the code and simple things like searching for conversion rules of a specific type become difficult.